optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc

```diff
@@ -0,0 +1,5 @@
+:quickstartsCloneUrl: https://github.com/kiegroup/optaplanner-quickstarts
+:quickstartsArchiveUrl: https://www.optaplanner.org/download/download.html
+:helloWorldJavaQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/hello-world
+:springBootQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/technology/java-spring-boot
+:quarkusQuickstartUrl: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
```

Comment:
Can we change these to follow the asciidoc guidelines style?
So quarkus-quickstart-url IIRC.

Reply:
meet.google.com/qur-fmqu-wav

Reply:
That probably applies to the maven/antora injected attributes too.

Should we mention those here too, in a comment? Or mention in which file(s) to find those?